src/test/java/dk/cngroup/lentils/service/ObjectGenerator.java

```diff
@@ -60,6 +63,24 @@ public Team generateTeam() {
         return teams;
     }
 
+    public List<Hint> generateHintList(List<Cypher> cyphers) {
+
+        List<Hint> hints = new LinkedList<>();
+
+        for (Cypher cypher : cyphers) {
+            IntStream.range(0, NUMBER_OF_HINTS_FOR_CYPHER).forEach(i -> hints.add(new Hint("text", 5, cypher)));
```

Comment:
Pokud se na tento radek podivas, tak co tento radek dela? Rozvedeme si to:
```java
for (Cypher cypher : cyphers) {
    IntStream.range(0, NUMBER_OF_HINTS_FOR_CYPHER) // Vytvor posloupnost cisel (konkrente od 0 do 3, takze 0, 1, 2, 3)
            .forEach( // pro kazde cislo <i> z toho stremu proved:
                    i -> // hlavicka lambdy: parametr <i>, ve kterem budou cisla z posloupnosti
                            hints.add( // Pridej do seznamu <hints>
                            new Hint( // Novy hint, ktery bude mit
                                    "text", //  <text>="text"
                                    5, // <value>=5
                                    cypher // <cypher>= aktualni cypher z forcyklu
                            )));
                    // !!! Otazka za zlateho bludistaka !!! Kde pouzivas aktualni cislo <i> ze stremu,
                    // ktere si nadefinovala v hlavicce lambdy?
}
````
Vedela by si odpovedet na otazku za zlateho bludistaka? :)
Kazdopadne si myslim, ze toto jednoduche reseni pres for cyklus muze byt ve finale mnohem citelnejsi:
```java
public List<Hint> generateHintList(List<Cypher> cyphers) {
    List<Hint> hints = new LinkedList<>();
    for (Cypher cypher : cyphers) {
        for (int i = 0; i < NUMBER_OF_HINTS_FOR_CYPHER; i++) {
            hints.add(new Hint("text", i, cypher));
        }
    }
    return hints;
}
````